pkg/controller/iotconfig/infinispan_device_registry.go

```diff
@@ -57,6 +57,7 @@ func (r *ReconcileIoTConfig) processInfinispanDeviceRegistry(ctx context.Context
 func (r *ReconcileIoTConfig) reconcileInfinispanDeviceRegistryDeployment(config *iotv1alpha1.IoTConfig, deployment *appsv1.Deployment) error {
 
 	install.ApplyDeploymentDefaults(deployment, "iot", deployment.Name)
+	deployment.Annotations["iot.registry.type"] = "infinispan"
```

Comment:
How about 'enmasse.io/iot-registry-type'? This matches the format we use for other annotation.

Reply:
I am using `iot.enmasse.io` now, as this is the groupId of the resource. I also created a constant for this string.